permissions-policy/resources/orientation-event.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+async function waitForOrientationEvent(eventName) {
+  if (eventName !== 'devicemotion' && eventName !== 'deviceorientation' && eventName !== 'deviceorientationabsolute') {
+    return 'ERROR';
```

Comment:
I wonder if we should throw real errors here. It might make things easier to debug if things go wrong.